tests/test_bench.py

```diff
@@ -35,13 +44,11 @@
 from pytket.extensions.qiskit import tk_to_qiskit
 from qiskit import QuantumCircuit
 
-test_qasm_output_path = "./test_output/"
+TEST_QASM_OUTPUT_PATH = "./test_output/"
```

Comment:
Why not make this a pytest fixture instead of a global variable?

Reply:
Implemented that.